.github/workflows/main.yml

```diff
@@ -32,6 +32,10 @@ jobs:
       working-directory: crates/usvg
       run: cargo check --no-default-features
 
+    - name: Build usvg with all features
```

Comment:
`usvg` will all features will be build during `resvg` testing.

Reply:
Removed